output/adc2vtk.pl

```diff
@@ -653,6 +798,40 @@
 printf XDMFFLUXBOUNDARY "   </Domain>\n";
```

Comment:
I know this particular line is existing code, but writing XML is a good thing to do using template toolkit - just an fyi, I am not suggesting you redo this as a tt

Reply:
I think I had a look at the Perl XML module back in the day when I first started working on this ... the documentation started off with "First, you have to carefully design a complete XML Schema ..." and I thought, oh no, I don't have time/inclination for that, I am just trying to write a simple little text file. And then of course once it worked for my simple test case, the whole thing got out of hand. :-) Yeah, I agree this is a pretty suboptimal way to go about it ...